communities/UNAM-Engineering/community-assessment.md

```diff
@@ -0,0 +1,162 @@
+# UNAM Engineering 
+[UNAM Engineering](https://github.com/unamfi) is a small Github organization started on 2016 with the objective to provide students a space
+where they were able to share the projects they have been developing on their classes and hackathons, tools and tips for other students. We
+promote students to go to meetups and various event, help them prepare for job/internship opportunities and provide office hours.
+
+### Definition
+
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - Students interested in developing projects in their free time.
+  - Students already working on research and/or developing apps.
+  - Students interested to attend hackathons and meetups.
+  - Students interested in landing internship opportunities.
+  - Students interested in meeting industry companies and people.
+
+  
+### Size
+- How many people are currently in your community?
+
+Currently there's 45 people on the GitHub organization and around 10 people have submitted repos.
+
+- How many would you _like_ to be in your community?
+
+Getting to 150 people in the organization and having a 20% - 25% members contributing.
+
+- Historically, has the group always been about the same size? If not, what brought on that change?
+
+The group grew a lot in the beginning and people used to share they're code. After four months the growth
+stopped because we didn't promote it outside of our core friends. Usually, the last couple months of the semester
+we get a lot of traffic from students looking for final projects done on previous years.
```

Comment:
That seasonality is very interesting; is there anything you can do proactively to support that activity and get new members from it? A final year project showcase or somethinjg like that